pychunkedgraph/app/meshing/common.py

```diff
@@ -33,7 +33,7 @@ def _remeshing(serialized_cg_info, lvl2_nodes):
 
     # TODO: stop_layer and mip should be configurable by dataset
     meshgen.remeshing(
-        cg, lvl2_nodes, stop_layer=4, cv_path=None, cv_mesh_dir=None, mip=1, max_err=40
+        cg, lvl2_nodes, stop_layer=4, cv_path=None, cv_mesh_dir=None, mip=1, max_err=320
```

Comment:
Would like to be consistent for mesh generation. Would it be okay to set this back to 40?

Reply:
I changed it to 320 because that is the simplification error I used for initial meshing in FAFB